### ISSUE TITLE
bug with list comprehension since 0.3

### ISSUE THREAD

**Issue by basnijholt** [custom-components/pyscript#32]
Updating to 0.3 I suddenly see this message in my error-logs
`SyntaxError: no binding for nonlocal 'entity_id' found`

Reproduce with:
```python
def white_or_cozy(group_entity_id):
    entity_ids = state.get_attr(group_entity_id)['entity_id']
    attrs = [state.get_attr(entity_id) for entity_id in entity_ids]
```

**Comment by dlashua** [comment]
If it helps with debugging, this code works fine:

```
def white_or_cozy(group_entity_id):
    entity_ids = state.get_attr(group_entity_id)['entity_id']
    attrs = []
    for entity_id in entity_ids:
        attrs.append(state.get_attr(entity_id))
```

**Comment by craigbarratt** [comment]
Sorry about that.  This got broken when scoping was fixed.  Fix c1e725f pushed.
